docs/core/windows-prerequisites.md

```diff
@@ -83,29 +82,6 @@ Once you have Visual Studio configured with .NET Core 2.1 SDK, you can do the fo
 * Retarget .NET Core 1.x and 2.0 projects to .NET Core 2.1, build, and run.
```

Comment:
The picture above shows .NET Core 2.0 - is it worth updating it?
Also, given that .NET Core 2.0 is default in VS 15.7, should we mention here that it is out of support?